docs/README.md

```diff
@@ -1,15 +1,39 @@
-# Building the documentation
+# Meson Documentation
 
-1. Get [hotdoc](https://hotdoc.github.io/installing.html) (0.8.9 required)
-1. Run hotdoc in the docs/ directory:
+## Build dependencies
 
-    ../meson/meson.py build/
+Meson uses itself and [hotdoc](https://github.com/hotdoc/hotdoc) for generating documentation.
+
+Minimum required version of hotdoc is *0.8.9*.
+
+Instructions on how to install hotdoc are [here](https://hotdoc.github.io/installing.html).
+
+## Building the documentation
+
+```
+$ cd meson/docs/
```

Comment:
This should either be `meson.git/docs` or just `docs/` because otherwise people might look for a `meson` subdir inside the git repo.